audio-worklet/bitcrusher-test.html

```diff
@@ -16,171 +16,184 @@
+                      maxError + ', exceeds the maximum permitted error, ' +
+                      permittedSampleError);
+
+              // Verify samples from unaltered oscillator match samples from
+              // bitcrushed oscillator with non-information reducing parameters.
+              for (let i = 0; i < originalOutput.length; i++) {
+                const crushedSample = bitcrusherOutput[i];
+                const originalSample = originalOutput[i];
+                const diff = Math.abs(originalSample - crushedSample);
+                if (i < bufferSize) {
+                  console.assert(
+                      bitcrusherOutput[i] === 0,
+                      'Bitcrusher sample at ' + i + 'expected to be 0 but it'
+                      + ' was ' + bitcrusherOutput[i]);
+                }
+                console.assert(
+                    diff < permittedSampleError,
+                    'Bitcrushed sample at ' + i + ' is ' + crushedSample + 
+                    ' but ' + originalSample + ' for oscillator');
+              }
+            });
+      }
+
+      /**
+       * Verify that bitcrusher handles illegal input correctly.
+       */
+      function testIllegalInput() {
+        try {
+          let bitcrusher = new Bitcrusher(0);
+          console.error('Bitcrusher did not throw illegal context error');
+        }
+        catch (error) {}
+      }
+
+      /**
+       * Verify that bitcrusher applies sample reduction on multiple channels of 
+       * input, and that sample reduction works across multiple buffers (which 
+       * requires bitcrusher memory.)
+       */
+      function testMultipleChannels() {
+        let channels = 2;
+        let context = new OfflineAudioContext(channels, 44100, 44100);
+        // Reduction should not be a factor of bufferSize for full coverage.
+        const reduction = 7; 
+        const bitDepth = 24;
+        const bufferSize = 512; 
+        // The two signals should differ to verify that the bitcrusher does not
+        // place the results from one channel to another.
+        let oscillator = new OscillatorNode(context, {frequency: 112});
+        let oscillator2 = new OscillatorNode(context, {frequency: 231});
+        let merger = new ChannelMergerNode(context, {numberOfInputs: 2});
+        let bitcrusher = new Bitcrusher(
+            context, {bufferSize, channels, bitDepth, reduction});
+        
+        oscillator.connect(merger, 0, 0);
+        oscillator2.connect(merger, 0, 1);
+        merger.connect(bitcrusher.input);
+        bitcrusher.output.connect(context.destination); 
+        oscillator.start();
+        oscillator2.start();
+
+        context.startRendering().then((buffer) => {  
+            let channelsDiffer = false;
+            for (let i = 0; i < buffer.getChannelData(0).length; i++) {
+              if (buffer.getChannelData(0)[i] != buffer.getChannelData(1)[i])
+                channelsDiffer = true;
```

Comment:
Might as well `break` after this, since it doesn't matter if there are more differences.